### ISSUE TITLE
New Pyramid scheme

### ISSUE THREAD

**Issue by romkapomka** [MetaMask/eth-phishing-detect#2340]
Website https://333eth.io . Please add to blacklist

**Comment by EtherBomb** [comment]
NOT SCAM!!!
The project 333eth.io is not a Scam it is a mistake!!! Thank U
It's competitors trying to do badly!!

**Comment by sashok76rus** [comment]
Сайт отличный. Все стабильно выплачивает! !!!!!!!
Закрывать его не за что.

**Comment by colorist71** [comment]
Competitors site deliberately slander .Understand and restore justice

**Comment by koka73** [comment]
333ETH.IO - IS NOT A SCAM AND NOT A FISHING SITE.
This is the machinations of competitors. Unlock this site, please

**Comment by Konstantin48** [comment]
333eth.io is not scam!!! This is a crypto game and people participate in it only on their own initiative!

**Comment by SergeyTit** [comment]
The project regularly pays all the promised payments. It's a shame when a complaint of one offended person can spoil the mood of a large community. Please remove the site 333eth.io from the blacklist. Thx.

**Comment by GooddayAlex** [comment]
Good site ! No scam... Хороший сайт и проект ! Все работает !

**Comment by igor51rus** [comment]
I ask you to remove the site 333eth.io from the blacklist. The site 333eth.io is entered by mistake in the black list MetaMask. Dear support, I am an investor of this site. Unblock the site 333eth.io and put everything back in place. The site 333eth.io is on the blacklist erroneously due to competitors.

**Comment by Barloch77** [comment]
Immediately unlock the site https: //www.333eth.io! You interfere with the work of a good project.

**Comment by TeamXXI** [comment]
This is not a scam! The contract regularly pays all its investors, please remove the restriction metamask on the site 333eth.io
It was added by mistake because of the competitive environment.
Thank you!

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/2374
